modules/xmpp/TraceablePeerConnection.js

```diff
@@ -322,11 +321,8 @@ TraceablePeerConnection.prototype.addStream = function (stream, ssrcInfo) {
     }
 };
 
-TraceablePeerConnection.prototype.removeStream = function (stream, stopStreams) {
+TraceablePeerConnection.prototype.removeStream = function (stream) {
     this.trace('removeStream', stream.id);
-    if (stopStreams) {
-        RTC.stopMediaStream(stream);
```

Comment:
the only place (in the code before my changes) that i see passing a boolean it at all is: https://github.com/jitsi/lib-jitsi-meet/blob/f7efba4aecda8fe6b3d6366a0d3bf1a054f85b8c/modules/xmpp/JingleSessionPC.js#L1031, so yeah i think we're ok